View/StrategyCtrView.py

```diff
@@ -247,18 +326,37 @@ def refresh_strat(self, strats):
         else:
             self.selectStrat.addItem('Aucune Stratégie disponible')
 
+    def send_quick_strat1(self):
+        if len(self.strat_default) > 0:
+            self.parent.model_dataout.send_strategy(self.strat_default[0], self.parent.get_team_color())
+
+    def send_quick_strat2(self):
+        if len(self.strat_default) > 1:
+            self.parent.model_dataout.send_strategy(self.strat_default[1], self.parent.get_team_color())
+
+    def send_quick_tactic1(self):
+        if len(self.tactic_default) > 0:
+            self.send_tactic(self.tactic_default[0])
+
+    def send_quick_tactic2(self):
+        if len(self.tactic_default) > 1:
+            self.send_tactic(self.tactic_default[1])
+
     def send_strat(self):
         strat = str(self.selectStrat.currentText())
         if not strat == 'Aucune Stratégie disponible':
             self.parent.model_dataout.send_strategy(strat, self.parent.get_team_color())
 
-    def send_tactic(self):
-        id_bot = int(self.selectRobot.currentText())
+    def send_apply_tactic(self):
         tactic = str(self.selectTactic.currentText())
+        if not tactic == 'Aucune Tactique disponible':
+            self.send_tactic(tactic)
+
+    def send_tactic(self, tactic: str):
+        id_bot = int(self.selectRobot.currentText())
         args = str(self.argumentsLine.text()).split()
         target = self.parent.model_dataout.target
-        if not tactic == 'Aucune Tactique disponible':
-            self.parent.model_dataout.send_tactic(id_bot, self.parent.get_team_color(), tactic=tactic, target=target, args=args)
+        self.parent.model_dataout.send_tactic(id_bot, self.parent.get_team_color(), tactic=tactic, target=target, args=args)
```

Comment:
args=args is not really clear. which informations are transmit?

Reply:
They are the optional argument field that's pass to the tactic. (The empty field over the "Appliquez'' button). For example, in GoToPosition the argument is the cruise speed.
It's one the stuff that I don't really like about the StrategyIA<=>UiDebug relation.